lib/index.d.ts

```diff
@@ -0,0 +1,600 @@
+import { OrderItem } from "./parsers/orderItems";
+
+type InitParams = {
+    accessKeyId: string;
+    authToken?: string;
+    host?: string;
+    merchantId: string;
+    port?: number;
+    region?: string;
+    secretAccessKey: string;
+};
+
+// cSpell: disable
+type MWS_MARKETPLACES = {
+    AU: 'A39IBJ37TRP1C6',
+    BR: 'A2Q3Y263D00KWC',
+    CA: 'A2EUQ1WTGCTBG2',
+    CN: 'AAHKV2X7AFYLW',
+    DE: 'A1PA6795UKMFR9',
+    ES: 'A1RKKUPIHCS9HS',
+    FR: 'A13V1IB3VIYZZH',
+    IN: 'A21TJRUUN4KGV',
+    IT: 'APJ6JRA9NG5V4',
+    JP: 'A1VC38T7YXB528',
+    MX: 'A1AM78C64UM0Y8',
+    TR: 'A33AVAJ2PDY3EV',
+    UK: 'A1F83G8C2ARO7P',
+    US: 'ATVPDKIKX0DER',
+};
+// cSpell: enable
+
+type ValueOf<T> = T[keyof T];
+type MarketplaceId = ValueOf<MWS_MARKETPLACES>;
+type CountryCode = keyof MWS_MARKETPLACES;
+
+type MARKET_CURRENCY = {
```

Comment:
Are these all the currencies that we need to support here?

Reply:
Probably not -- anyone know if there's a list of all supported currencies on Amazon's side?  When I went and zipped through *all* the documentation while writing https://github.com/ericblade/ts-amazon-mws I seem to recall noticing that only a handful of currency codes were mentioned, which I thought seemed strange.

Reply:
noted to update also on #179 .. if someone wants a type that lists all possible currencies they can use @types/dinero.js but this is just the default currency code for each marketplace which is going to be a much smaller list.  just have to create that list is all.